package-lock.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
This was a package-lock.json at the root of the project, probably because someone ran an `npm` command in the root of the project :wink: